migrator/migrations/m_192_to_m_193_report_config_migrations/schema/report_snapshots.go

```diff
@@ -0,0 +1,60 @@
+// Code generated by pg-bindings generator. DO NOT EDIT.
+
+package schema
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/walker"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/search"
+)
+
+var (
+	// CreateTableReportSnapshotsStmt holds the create statement for table `report_snapshots`.
+	CreateTableReportSnapshotsStmt = &postgres.CreateStmts{
+		GormModel: (*ReportSnapshots)(nil),
+		Children:  []*postgres.CreateStmts{},
+	}
+
+	// ReportSnapshotsSchema is the go schema for table `report_snapshots`.
+	ReportSnapshotsSchema = func() *walker.Schema {
+		schema := walker.Walk(reflect.TypeOf((*storage.ReportSnapshot)(nil)), "report_snapshots")
+		referencedSchemas := map[string]*walker.Schema{
```

Comment:
`referencedSchemas` and the `ResolveReferences` need to be removed.  TIL those will cause issues if down the road we add a new reference.  Those are only used to utilize the search framework and we are not and should not use that in a migration.  Which as an aside, may make it slightly more difficult to use stores now that the generic store uses the search framework.